standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1087,6 +1096,10 @@ public void abortTxn(AbortTxnRequest rqst) throws NoSuchTxnException, MetaExcept
   @RetrySemantics.Idempotent
   public void abortTxns(AbortTxnsRequest rqst) throws MetaException {
     List<Long> txnIds = rqst.getTxn_ids();
+    TxnErrorMsg txnErrorMsg = TxnErrorMsg.NONE;
```

Comment:
in the above method (abortTxn(AbortTxnRequest rqst)), default value was ABORT_ROLLBACK and here it's NONE. Is this expected?

Reply:
Changed default values in all places to `TxnErrorMsg.NONE`. Done.